ios/app/MBXAppDelegate.m

```diff
@@ -1,11 +1,27 @@
 #import "MBXAppDelegate.h"
 #import "MBXViewController.h"
+#import <mbgl/ios/MapboxGL.h>
 #import <mbgl/ios/MGLMapboxEvents.h>
 
 @implementation MBXAppDelegate
 
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
 {
+    // Set Access Token
+    NSString *accessToken = [[NSProcessInfo processInfo] environment][@"MAPBOX_ACCESS_TOKEN"];
+    if (accessToken) {
+        // Store to preferences so that we can launch the app later on without having to specify
+        // token.
+        [[NSUserDefaults standardUserDefaults] setObject:accessToken forKey:@"access_token"];
```

Comment:
We'd be better off putting things in a constant to avoid future potential problems with the `@"access_token"` [magic string](http://www.cimgf.com/2013/01/29/down-with-magic-strings/), too.